src/unix.rs

```diff
@@ -81,37 +82,41 @@ impl Client {
         Ok(Client::from_fds(pipes[0], pipes[1]))
     }
 
-    pub unsafe fn open(s: &str, check_pipe: bool) -> io::Result<Client> {
+    pub unsafe fn open(s: &str, check_is_pipe: bool) -> Result<Client, FromEnvError> {
```

Comment:
```suggestion
    pub unsafe fn open(s: &str, check_pipe: bool) -> Result<Client, FromEnvError> {
```
Please use `check_pipe` everywhere, `check_is_pipe` just doesn't read well.